### DISCUSSION TITLE
Customize vibrations on hour and half hour (avoid nights/weekends)

### DISCUSSION THREAD

**Discussion by famiclone6502** [InfiniTimeOrg/InfiniTime#1617] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

Only chime on weekdays and during the day

### Description

Chimes help break my focus during work hours and check for meetings I need to attend. Can you add a menu option to start and stop these during custom hours? Also, a check box for whether or not they occur only weekdays (Monday through Friday)? Otherwise it will keep me awake at night, and I do not need them during the weekend. Right now I need to disable the chimes manually on my PineTime.

I've been using a Timex Data-Link with a custom Motorola 6805 application to perform chimes on a specific schedule. Right now it does hourly chimes during the day (9 AM to - 6 PM) and only during weekdays. I do appreciate the half-hour chimes that the PineTime does already, which I wanted on the Timex.

Ideally, customization could eventually go even further. There's a lot of history in different types of chimes clocks/watches have offered like different repeaters and sonneries. Personally, I would like an additional reminder a minute before the hour or half-hour. Customizing with a shorter vibration for that, plus maybe an audible chime on the hour would be helpful. Others may also only want specific days of the week. These are not crucial to me but would be nice to consider in the design of related features. 

This request builds upon issue #778, which added the option for vibration on the hour and half hour. Without this feature I would not have bought the PineTime, so thank you for fulfilling this request.

**Comment by minacode** [comment]
> Otherwise it will keep me awake at night

Sleep mode disable chimes. Maybe that it worth a try for you.

**Comment by famiclone6502** [comment]
> Sleep mode disable chimes. Maybe that it worth a try for you.

From what I understand, it is still a manual process to enter sleep mode. Either I manually enter sleep mode, or I manually disable the chimes. Automatic sleep mode would be a welcome feature, but I also still want other notifications outside of work hours (i.e. during the weekend), just not chimes.

**Comment by Boteium** [comment]
Automatic sleep mode sound awesome. It can also save battery during night.

**Comment by Riksu9000** [comment]
I've converted this issue to an idea discussion, since it's a feature request, but there seem to be underlying issues to this request, so feel free to open a new issue describing the issues more specifically using the issue form.

**Comment by agittins** [comment]
Would expanding the bluetooth api so that companion apps could control the settings be a workable solution? Then we could automate it via the alarms/modes set on our phones.

I'd really like almost *every* setting to be controllable by bluetooth, so that my companion app (and ultimately home-assistant) can alter settings on the watch based on variables outside of the watch (chimes, sleep mode, alarms, wakeup options etc).

In my particular case I have irregular sleep patterns, so setting sleep mode for me happens at different times every day. I also want to set timers / notifications based on time elapsed since waking, which is also best done outside of the watch.

I think that people will have such varying needs of how they want the watch to behave, that expanding the bluetooth api so that companion apps can have more control over the device might be the best way to fit that while still adhering to the infinitime vision. Given the watch has limited resources, having companion apps take on the customisation might be a good path to giving users flexibility without making the UI on the watch more complicated.
